proto/proto-backplane-grpc/Dockerfile

```diff
@@ -24,7 +24,8 @@ RUN set -eux; \
     /includes/deephaven/proto/table.proto \
     /includes/deephaven/proto/application.proto \
     /includes/deephaven/proto/inputtable.proto \
-    /includes/deephaven/proto/partitionedtable.proto; \
+    /includes/deephaven/proto/partitionedtable.proto \
+    /includes/grpc/health/v1/health.proto; \
```

Comment:
why do we need to generate java/js/go/py for this file? If we include grpc-services, then the generated java at least should already be present?

Reply:
I'll try it again w/o the extra proto...